src/main/java/org/ibp/api/rest/inventory_new/LotResource.java

```diff
@@ -67,34 +70,43 @@
 							"Multiple sort criteria are supported.")
 	})
 	@ResponseBody
-	public ResponseEntity<List<LotDto>> getLots(@PathVariable final String cropName, //
+	public ResponseEntity<List<ExtendedLotDto>> getLots(@PathVariable final String cropName, //
 		@RequestParam final Integer searchRequestId, @ApiIgnore
 	final Pageable pageable) {
 
 		final LotsSearchDto searchDTO = (LotsSearchDto) this.searchRequestService
 			.getSearchRequest(searchRequestId, LotsSearchDto.class);
 
-		final PagedResult<LotDto> resultPage =
-			new PaginatedSearch().executeBrapiSearch(pageable.getPageNumber(), pageable.getPageSize(), new SearchSpec<LotDto>() {
+		final PagedResult<ExtendedLotDto> resultPage =
+			new PaginatedSearch().executeBrapiSearch(pageable.getPageNumber(), pageable.getPageSize(), new SearchSpec<ExtendedLotDto>() {
 
 				@Override
 				public long getCount() {
 					return LotResource.this.lotService.countSearchLots(searchDTO);
 				}
 
 				@Override
-				public List<LotDto> getResults(final PagedResult<LotDto> pagedResult) {
+				public List<ExtendedLotDto> getResults(final PagedResult<ExtendedLotDto> pagedResult) {
 					return LotResource.this.lotService.searchLots(searchDTO, pageable);
 				}
 			});
 
-		final List<LotDto> lotDtos = resultPage.getPageResults();
+		final List<ExtendedLotDto> extendedLotDtos = resultPage.getPageResults();
 
 		final HttpHeaders headers = new HttpHeaders();
 		headers.add("X-Total-Count", Long.toString(resultPage.getTotalResults()));
 
-		return new ResponseEntity<>(lotDtos, headers, HttpStatus.OK);
+		return new ResponseEntity<>(extendedLotDtos, headers, HttpStatus.OK);
 
 	}
 
+	@ApiOperation(value = "Create Lot with Initial Transaction", notes = "Create a new lot and its initial transaction")
```

Comment:
@vmaletta please remove with Initial transaction from value, notes and apiparam

Reply:
ok